cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureResourceConnector.java

```diff
@@ -141,6 +141,24 @@
         return resources;
     }
 
+    private List<CloudResource> persistCloudResources(
+            AuthenticatedContext ac, CloudStack stack, PersistenceNotifier notifier, CloudContext cloudContext, String stackName,
+            String resourceGroupName, Deployment templateDeployment) {
+        List<CloudResource> allResourcesToPersist = new ArrayList<>();
+        List<CloudResource> instances;
+        try {
+            List<CloudResource> templateResources = azureCloudResourceService.getDeploymentCloudResources(templateDeployment);
```

Comment:
We should add log messages to this method about what's happening.

Reply:
Yes, I added some messages.